src/NuGet.Services.AzureSearch/SearchService/SearchTextBuilder.cs

```diff
@@ -12,7 +12,7 @@ namespace NuGet.Services.AzureSearch.SearchService
 {
     public partial class SearchTextBuilder : ISearchTextBuilder
     {
-        private const string MatchAllDocumentsQuery = "*";
+        public static string MatchAllDocumentsQuery = "*";
```

Comment:
This should not be public I think. Instead `IndexOperationBuilder` should detect an empty `ParsedQuery.Grouping` for this.